gem2rpm/scl.spec.erb

```diff
@@ -6,12 +6,15 @@ SCL_PREFIXES = JSON.load(File.read(File.join(root, 'scl_prefixes.json')))
 LICENSES = JSON.load(File.read(File.join(root, 'licenses.json')))
 
 config.rules[:doc] << /\/?CHANGES/
+config.rules[:doc] << 'web'
```

Comment:
I still feel like web (and sample below) are too generic and have a risk of causing problems in the future. @evgeni?

Reply:
I am not versed enough in the standard (or well, in this case less standard) layouts of gems and what can be hidden in these folders.

That said, the content of the `web` folder doesn't look like docs directly -- these are templates that need processing and look kinda useless w/o it. So for `racc`, I lean towards `%exclude`ing it completely.

And then I know no examples of `web` being anything, so maybe it should not be part of the template either?

`sample` on the other hand actually looks like useful examples, which I'd consider documentation? But the `sample` name is indeed a tad off, I'd usually put those in `examples`.